#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_packinfo_txt.dpatch by Thibaut Paumard <paumard@users.sourceforge.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove yorz-specific code.
## DP: discuss it with upstream.

@DPATCH@

--- yorick-2.1.04+dfsg+cvs20071118.orig/doc/html/packinfo.txt	2007-11-21 16:52:06.000000000 +0100
+++ yorick-2.1.04+dfsg+cvs20071118/doc/html/packinfo.txt	2007-11-21 16:52:40.000000000 +0100
@@ -15,10 +15,6 @@
 :newsection
 Compiled (plugin) Yorick Packages
 
-:yorz
-read and write png or jpeg grephics files, write mpeg-1 movie files,
- and interface to zlib (gzip) compression library
-
 :drat
 a 2D transport equation solver for (r,z) meshes
 
